Core/include/Acts/Utilities/MathHelpers.hpp

```diff
@@ -12,6 +12,50 @@
 
 namespace Acts {
 
+/// @brief Returns the absolute of a number
+///        (Can be removed for c++ 23)
+template <typename T>
+constexpr T abs(const T n) {
```

Comment:
I'm not sure I understand this one. What does this add over `std::abs`? Are you expecting to use this with types that are not supported in `std::abs`?